<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LookupRaster</class>
 <widget class="QMainWindow" name="LookupRaster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lookup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="mainGridLayout" rowstretch="0,0,0,0,0,0,0,0" columnstretch="2,1,1">
      <item row="0" column="0">
       <widget class="QLabel" name="rasterLabel">
        <property name="text">
         <string>Raster</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="rasterToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="rasterComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lookupFieldLabel">
        <property name="text">
         <string>Lookup Fields</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="lookupFieldComboBox"/>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="ratCheckBox">
        <property name="text">
         <string>Append attribute table</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="outRasterLabel">
        <property name="text">
         <string>Output lookup raster</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QToolButton" name="outRasterToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLineEdit" name="outRasterLineEdit"/>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="applyPushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>rasterComboBox</tabstop>
  <tabstop>rasterToolButton</tabstop>
  <tabstop>lookupFieldComboBox</tabstop>
  <tabstop>ratCheckBox</tabstop>
  <tabstop>outRasterLineEdit</tabstop>
  <tabstop>outRasterToolButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
  <tabstop>applyPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
